backend/battleship.py

```diff
@@ -93,13 +110,14 @@ def player_move(self, value):
         arguments = {
             'move_id': value['id'],
             'game_id': value['gameId'],
-            'result': self.calculate_result(guess=value['move'])
+            'result': self.calculate_result(guess=value['coordinate']),
+            'proof': zokrates_move_validator(self._zokrates_executable_location, self.ships_positions, parse(self.proof).get("proof")) # TODO Change attribute in get method to real hash
```

Comment:
`# TODO Change attribute in get method to real hash`
is this TODO item still valid?